### ISSUE TITLE
Add support for `postcss-scss` in `declaration-nested-properties-no-divided-groups` rule

### ISSUE THREAD

**Issue by jhae-de** [stylelint-scss/stylelint-scss#1262]
I’ve always wondered why the example [documented](https://github.com/stylelint-scss/stylelint-scss/tree/v7.0.0/src/rules/declaration-nested-properties-no-divided-groups) in the `declaration-nested-properties-no-divided-groups` rule doesn’t trigger an error when using the `stylelint-config-standard-scss` configuration, even though the rule is enabled there (specifically in the `stylelint-config-recommended-scss` configuration):

```scss
a {
  background: url(img.png) center {
    size: auto;
  }
  background: {
    repeat: no-repeat;
  }
}
```

I finally had some time to investigate the cause. It turns out that the `stylelint-config-recommended-scss` configuration sets `postcss-scss` as custom syntax, but the rule doesn’t support that syntax at all. You can reproduce this by adding `customSyntax: "postcss-scss"` to the `reject` tests. In that case, all of these tests fail.

**Comment by kristerkari** [comment]
Nice, good catch!
